lib/icinga/externalcommandprocessor.cpp

```diff
@@ -970,10 +971,19 @@ void ExternalCommandProcessor::ScheduleSvcDowntime(double, const std::vector<Str
 void ExternalCommandProcessor::DelSvcDowntime(double, const std::vector<String>& arguments)
 {
 	int id = Convert::ToLong(arguments[0]);
-	Log(LogNotice, "ExternalCommandProcessor")
-		<< "Removing downtime ID " << arguments[0];
+	m_DownTimeRemoved = false;
 	String rid = Downtime::GetDowntimeIDFromLegacyID(id);
-	Downtime::RemoveDowntime(rid, true);
+
+	try {
+		Downtime::RemoveDowntime(rid, true);
+	} catch (std::runtime_error& error) {
+		m_DownTimeRemoved = true;
```

Comment:
This reads like either the value or the name should be inverted. You set `removed = true` if removing failed.